.circleci/config.yml

```diff
@@ -1023,6 +1060,44 @@ jobs:
       - store_test_results:
           path: test/test-results/e2e
 
+  test-e2e-mmi-playwright - OPTIONAL:
+    executor: playwright
+    parallelism: 2
+    steps:
+      - run: *shallow-git-clone
+      - attach_workspace:
+          at: .
+      - run:
+          name: Move test build to dist
+          command: mv ./dist-test-mmi-playwright ./dist
+      - run:
+          name: Install chromium
+          command: |
+            yarn playwright install chromium
+      - run:
+          name: test:e2e:chrome:mmi
+          command: |
+            SHARD="$((${CIRCLE_NODE_INDEX}+1))"; xvfb-run yarn playwright test --shard=${SHARD}/${CIRCLE_NODE_TOTAL}
+          no_output_timeout: 20m
```

Comment:
This is fine for now, but would be cool to see if we can somehow use the circleci commands to split tests so we can benefit from the "rerun failed tests" command that only reruns failed tests versus failed jobs. Not a blocker for this PR

Reply:
I didn't know that feature, nice! I'll create a ticket on my backlog to add it after the PR is merged